CHANGELOG.md

```diff
@@ -49,7 +49,7 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 
 ### New
 
-- TODO ([#XXX](https://github.com/kedacore/keda/issues/XXX))
+- **Governance**: KEDA transitioned to CNCF Graduated project ([press release](https://www.cncf.io/announcements/2023/08/22/cloud-native-computing-foundation-announces-graduation-of-kubernetes-autoscaler-keda/))
```

Comment:
I think that this entry will fail on changelog style check xD

Reply:
Or yes, but IDK why because I thin, this doesn't match the expected format 🤔 
Let me check it locally to ensure that we won't break the main branch 🙏

Reply:
yes, it fails locally:
![image](https://github.com/kedacore/keda/assets/36899226/ee9b687f-2c40-45cd-90a7-93811f3674de)

I have to review why is hasn't failed, but in the meantime, can we add the governance issue instead of the press release?

Reply:
The link (the caption and the link itself).
[The style ](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#changelog)says that the format has to be `<Scaler Name / General>: <Description> (#<ID>)` and the link has to be related with the given ID `https://github.com/kedacore/keda/issues/ISSUE`.
Do to that, the validation script checks the link format and also validates that the given ID is present in the link: 
https://github.com/kedacore/keda/blob/main/hack/validate-changelog.sh#L43

In this case, the ID isn't valid but the link neither

Reply:
This PR enables again the validation: https://github.com/kedacore/keda/pull/4901
Once it's merged, this entry will produce a validation failure

Reply:
Wouldn't it be better to relax the link validation? Does it have to be that strict? What if we want to link to a PR on another repo?

Reply:
Given the amount of style errors that I fixed for passing the check, I think that the check is worth as it is, but if you think that we can relax it, it's just a regex, we can update it :)

Reply:
Sounds fair, I was just wondering!

```suggestion
- **Governance**: KEDA transitioned to CNCF Graduated project ([press release](https://github.com/kedacore/governance/issues/63))
```